cmip6_downscaling/methods/common/tasks.py

```diff
@@ -161,6 +162,32 @@ def pyramid(
     return target
 
 
+@task
+def regrid(source_path: UPath, target_grid_path: UPath, run_parameters: RunParameters) -> UPath:
+
+    target = (
+        intermediate_dir
+        / "regrid"
+        / "_{model}_{scenario}_{variable}_{latmin}_{latmax}_{lonmin}_{lonmax}_{train_dates[0]}_{train_dates[1]}_{predict_dates[0]}_{predict_dates[1]}".format(
```

Comment:
let's create this part of the path as the combination of the `source_path` and `target_grid_path`. I don't think we need `run_parameters` to make this path.